src/views/navigation/directMessagesTab.js

```diff
@@ -76,12 +76,19 @@ const DirectMessagesTab = (props: Props) => {
     window.interop.setBadgeCount(unseenCount);
   }
 
+  const isWideViewport =
+    window && window.innerWidth > MIN_WIDTH_TO_EXPAND_NAVIGATION;
```

Comment:
This will break SSR rehydration any logged-in page I think? We should do this with CSS media queries inside the Tooltip component.